libraries/AP_Mount/AP_Mount_config.h

```diff
@@ -44,3 +44,7 @@
 #ifndef HAL_MOUNT_XACTI_ENABLED
 #define HAL_MOUNT_XACTI_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED && HAL_ENABLE_DRONECAN_DRIVERS
 #endif
+
+#ifndef HAL_MOUNT_VIEWPRO_ENABLED
+#define HAL_MOUNT_VIEWPRO_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED
```

Comment:
```suggestion
#define HAL_MOUNT_VIEWPRO_ENABLED AP_MOUNT_BACKEND_DEFAULT_ENABLED && BOARD_FLASH_SIZE > 1024
```

```
[874/879] Linking build/MatekF405-TE-bdshot/bin/arduplane
/home/pbarker/gcc/gcc-arm-none-eabi-10-2020-q4-major/bin/../lib/gcc/arm-none-eabi/10.2.1/../../../../arm-none-eabi/bin/ld:common.ld:202 cannot move location counter backwards (from 0000000008100340 to 0000000008100000)
collect2: error: ld returned 1 exit status
```

Reply:
ok, modified to only enable on higher flash boards.